fireapp/lib/presentation/constraint_form/constraint_form_view.dart

```diff
@@ -0,0 +1,132 @@
+import 'package:fireapp/presentation/constraint_form/date_input_field.dart';
+import 'package:fireapp/presentation/constraint_form/text_input_field.dart';
+import 'package:fireapp/presentation/constraint_form/time_input_field.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:fireapp/presentation/constraint_form/constraint_form_view_model.dart';
+import 'package:fireapp/presentation/constraint_form/base_input_field.dart';
+
+class SchedulerConstraintPage extends StatelessWidget {
+  const SchedulerConstraintPage({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(title: const Text('Scheduler')),
+      resizeToAvoidBottomInset: false,
+      body: const SchedulerConstraintForm(),
+    );
+  }
+}
+
+class SchedulerConstraintForm extends StatefulWidget {
+  const SchedulerConstraintForm({super.key});
+
+  @override
+  _SchedulerConstraintFormState createState() => _SchedulerConstraintFormState();
+}
+
+class _SchedulerConstraintFormState extends State<SchedulerConstraintForm> {
+  final viewModel = SchedulerConstraintFormViewModel();
+
+  @override
+  Widget build(BuildContext context) {
+    return Form(
+      key: viewModel.formKey,
+      child: Padding(
+        padding: const EdgeInsets.all(12.0),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            SchedulerTextInputField(
+              controller: viewModel.titleController,
+              style: Theme.of(context)
+                  .textTheme
+                  .labelLarge
+                  ?.copyWith(color: Colors.grey[700]),
+              label: AppLocalizations.of(context)?.volunteer_name ??
+                  "", // This will show volunteer name
+              icon: Icons.title,
+              validator: (v) => v!.isEmpty ? 'Title is empty!' : null,
+            ),
+            const SizedBox(height: 2.0),
```

Comment:
Rather than using multiple SizedBox, you should call `spacedBy()` on the array of Column children.